bugbug/models/stepstoreproduce.py

```diff
@@ -92,5 +92,20 @@ def get_labels(self):
 
         return classes, [0, 1]
 
+    def overwrite_classes(self, bugs, classes, probabilities):
+        for i, bug in enumerate(bugs):
+            if bug["cf_has_str"] == "no":
+                classes[i] = 0 if not probabilities else [1.0, 0.0]
+            elif bug["cf_has_str"] == "yes":
+                classes[i] = 1 if not probabilities else [0.0, 1.0]
+            elif "stepswanted" in bug["keywords"]:
+                classes[i] = 0 if not probabilities else [1.0, 0.0]
+            else:
```

Comment:
Let's remove this part about the history, as it's too slow compared to the other, quicker, checks.

Reply:
Now that I think of it, wouldn't bugs that don't fall into the first three categories obviously fall into this category? So we can just replace it with an else statement.

Reply:
Oh, maybe not. My bad.